notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>If we will not apply latexify() always then this cell is also redundant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>